ecosystem/platform/server/app/controllers/leaderboard_controller.rb

```diff
@@ -5,15 +5,15 @@
 
 class LeaderboardController < ApplicationController
   IT1_METRIC_KEYS = %i[rank validator liveness participation latest_reported_timestamp].freeze
+  IT1_RESULTS = File.read(File.join(Rails.root, 'public/it1_leaderboard_final.json'))
   It1Metric = Struct.new(*IT1_METRIC_KEYS)
 
   def it1
     expires_in 1.minute, public: true
     default_sort = [[:participation, -1], [:liveness, -1], [:latest_reported_timestamp, -1]]
     @metrics, @last_updated = Rails.cache.fetch(:it1_leaderboard, expires_in: 1.minute) do
-      response = HTTParty.get(ENV.fetch('LEADERBOARD_IT1_URL'))
-      metrics = JSON.parse(response.body).map do |metric|
-        timestamp = metric['latest_reported_timestamp'] ? DateTime.parse(metric['latest_reported_timestamp']).to_f : nil
+      metrics = JSON.parse(IT1_RESULTS).map do |metric|
+        timestamp = metric['latest_reported_timestamp']&.blank? ? nil : DateTime.parse(metric['latest_reported_timestamp']).to_f
```

Comment:
I don't remember ruby syntax completely, but isn't this & misplaced?  Otherwise the PR is all good

Reply:
This is Ruby's optional chaining operator. I just added it so that it doesn't blow up if metric.latest_reported_timestamp is null / not present.

Reply:
I see you're right though, if it is null / not present, the second clause will be evaluated (not what we want). I'll remove the optional chaining since it's not needed for the data that we have.